app/views/hyrax/base/_citations.html.erb

```diff
@@ -2,10 +2,14 @@
     <% if Hyrax.config.citations? %>
       <%= link_to "Citations", hyrax.citations_work_path(presenter), id: 'citations', class: 'btn btn-default' %>
     <% end %>
-    <div class='citations-heading'><%= t('.header') %></div>
```

Comment:
Please remove `.citations-heading` and `.citations-list` from `app/assets/stylesheets/hyrax/_work-show.scss`

Reply:
Will do